pkg/gen/filters/filterue/filters.go

```diff
@@ -14,4 +14,5 @@ func PopulateFuncMap(fm template.FuncMap) {
 	fm["ueType"] = ueType
 	fm["ueVar"] = ueVar
 	fm["ueVars"] = ueVars
+    fm["isStdSimpleType"] = isStdSimpleType
```

Comment:
Please prefix with _ue_. With the current setup all filters share the same namespace and we want to be clear where this filter is supposed to be used.